source/winVersion.py

```diff
@@ -124,21 +136,5 @@ def isUwpOcrAvailable():
 	return os.path.isdir(UWP_OCR_DATA_PATH)
 
 
-WIN10_RELEASE_NAME_TO_BUILDS = {

```

Comment:
This is part of the API, it can be deprecated, but shouldn't be removed.

Reply:
You'll need to restore this dictionary and add a comment that it is deprecated to be removed in 2022.1. You can also add a comment that it isn't used internally and should not be updated further.

Reply:
Could you make this happen automatically, like this:
``` python
# Deprecated: Windows 10 releases will be obtained from Windows Registry, no entries will be added.
# WIN10_RELEASE_NAME_TO_BUILDS will be removed in 2022.1.
if version_year < 2022:
	WIN10_RELEASE_NAME_TO_BUILDS = {
```